.github/workflows/nix.yaml

```
@@ -88,7 +75,28 @@
       docker-image-name: ${{ steps.docker.outputs.image_name }}
     
     steps:
+      - name: Check matrix conditions
+        id: check-conditions
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:32: Double quote to prevent globbing and word splitting

(shellcheck)